mmr-lib/Enemies.cs

```diff
@@ -19,7 +19,7 @@ public class ValueSwap
         public static void ReadEnemyList()
         {
             EnemyList = new List<Enemy>();
-            string[] lines = Properties.Resources.ENEMIES.Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
+            string[] lines = Resources.GetTextFile("ENEMIES.txt").Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
```

Comment:
Yes. In general, I think most people will avoid touching them out of fear of breaking the program. I also haven't heard of it being a huge problem in OoTR, even though most releases have always distributed the entire source code because the program is coded in Python (though I also don't pay attention to the support channels at all, so I could be wrong on that).

Still, i could add it back into one if it's an issue